commands/backfill.js

```diff
@@ -2,6 +2,8 @@ var tb = require('timebucket')
   , crypto = require('crypto')
   , objectifySelector = require('../lib/objectify-selector')
   , collectionService = require('../lib/services/collection-service')
+  , debug = require('../lib/debug')
```

Comment:
not used?

Reply:
oh no, sorry, i'm using it locally but I didn't remove it from the commit, can you do it during the merge?